src/states/bootstrapping.rs

```diff
@@ -49,19 +49,18 @@ pub enum TargetState {
 
 // State of Client or Node while bootstrapping.
 pub struct Bootstrapping {
```

Comment:
This commit (simplify state conversions) is officially my favourite commit of the day so far :smile: